autotest/emulator_tests.py

```diff
@@ -15,6 +15,8 @@
 mou_exe_path = get_exe_path("pestpp-mou")
 
 def dsi_freyberg(tmp_d,transforms=None,tag=""):
+    if not ies_exe_path:
+        pytest.skip("missing ies_exe_path")
```

Comment:
assume we will throw a fail elsewhere in the CI if bins are missing -- might be worth checking this.